### PR TITLE
Add migrate to tiers troubleshooting doc

### PR DESCRIPTION
**Pull request by andreidan**
This adds a troubleshooting doc for indices that mix index filtering allocation
with data tiers routing.

### CODE REVIEW

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,224 @@
+// tag::cloud[]
+In order to get the shards assigned we'll need to call the 
+<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
+resolve the conflicting routing configurations towards using the standardised
```
Comment (dakrone) [inline docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:4]:
```suggestion
In order to get the shards assigned we need to call the 
<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
resolve the conflicting routing configurations towards using the standardized
```

(don't blame me, blame the Elastic style guide for proscribing American spellings for things)

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
```
Comment (dakrone) [inline docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:18]: The "the name of your deployment" is disabled? Is that a type, or can the name of something be disabled?

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+// TESTRESPONSE[skip:didn't wait to stop it]
+
+//end::kibana-api-ex[]
+// end::cloud[]
+
+// tag::self-managed[]
+In order to get the shards assigned we'll need to make sure the deployment is 
+using the <<data-tiers,data tiers>> node roles and then call the 
+<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
+resolve the conflicting routing configurations towards using the standardised
```
Comment (dakrone) [inline docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:120]:
```suggestion
In order to get the shards assigned we need to make sure the deployment is 
using the <<data-tiers,data tiers>> node roles and then call the 
<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
resolve the conflicting routing configurations towards using the standardized
```

File: docs/reference/troubleshooting/data/data-tiers-mixed-with-node-attr.asciidoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
+[[troubleshoot-migrate-to-tiers]]
+== Indices mix index allocation filters with data tiers node roles to move through data tiers
+
+Elasticsearch standardised the implementation of https://www.elastic.co/blog/elasticsearch-data-lifecycle-management-with-data-tiers[hot-warm-cold architectures]
```
Comment (dakrone) [inline docs/reference/troubleshooting/data/data-tiers-mixed-with-node-attr.asciidoc:4]:
```suggestion
Elasticsearch standardized the implementation of https://www.elastic.co/blog/elasticsearch-data-lifecycle-management-with-data-tiers[hot-warm-cold architectures]
```

File: docs/reference/troubleshooting/data/data-tiers-mixed-with-node-attr.asciidoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+to <<data-tiers, data tiers>> in version 7.10.
+Some indices and deployments might have not fully transitioned to <<data-tiers, data tiers>>
+and mix the new way of implemetning the hot-warm-cold architecture with the 
+<<shard-allocation-filtering, legacy way>> based on node attributes.
```
Comment (dakrone) [inline docs/reference/troubleshooting/data/data-tiers-mixed-with-node-attr.asciidoc:8]:
```suggestion
and mix the new way of implementing the hot-warm-cold architecture with 
<<shard-allocation-filtering, legacy>> based node attributes.
```

**Review by dakrone — APPROVED** [review]
LGTM, I left some really *really* minor comments

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc (hunk 2 above)
Comment (jbaiera) [reply docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:18]: Maybe we just change this to "If your deployment is disabled" instead? It reads just a little strange to me. Might just be me though.

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support],
+or your deployment doesn't include {kib}, in which case all you need to do is
```
Comment (jbaiera) [inline docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:20]: Run on sentence: Should split the sentence into two. End the first after "please contact Support.", Start the new sentence with "If your deployment doesn't include Kibana, then all you need to do ...."

**Review by jbaiera — APPROVED** [review]
LGTM with some minor edits/suggests

File: docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc (hunk 6 above)
Comment (andreidan) [reply docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc:20]: Yes, good spot. Will do this in a follow-up for all guides. Thanks for the suggestion

**Pull request merged**